utility/servicer/module.go

```diff
@@ -182,31 +356,107 @@ func (s servicer) admitRelay(relay *coreTypes.Relay) error {
 
 	height := s.GetBus().GetConsensusModule().CurrentHeight()
 	if err := s.validateRelayMeta(relay.Meta, int64(height)); err != nil {
-		return fmt.Errorf("%w: %s", errValidateRelayMeta, err.Error())
+		return fmt.Errorf("%s: %w", err.Error(), errValidateRelayMeta)
 	}
 
 	// TODO: update the CLI to include ApplicationAddress(or Application Public Key) in the RelayMeta
```

Comment:
Seems like this TODO is no longer relevant? If not, remove.

Reply:
Thanks, fixed.